packages/mwp-app-render/src/util/dateTimeUtils.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Pads numbers less than two digits with an additional
+ * leading zero. Used for date and time formatting.
+ * @param {Number} num  Number to be padded
+ * @returns {String}    Returns a padded string
+ */
+export const padNumber = num => (num < 10 ? `0${num}` : `${num}`);
+
+/**
+ * Returns a formatted time string from the local value of the datetime
+ * object passed in for usage with the `find/upcoming_events` endpoint.
+ * The formatted date must follow the format: `HH:MM:SS`. Javascript's
+ * `.toISOString` method returns the iso value while we need the local value.
+ * @param {Object} timeObj  Datetime object
+ * @returns {String}        Formatted local date string in format `HH:MM:SS`
+ */
+export const getFormattedTime = timeObj =>
+        timeObj.toLocaleTimeString('UTC', { hour12: false });
```

Comment:
a nitpick, but since I'm new -- it looks like this is a lot of space for the indent. Is this the convention for the platform codebase?

Reply:
Ah, I think this is just my bad.